mkosi/mkosi.conf.d/debian-ubuntu/mkosi.conf

```diff
@@ -12,15 +12,12 @@ VolatilePackages=
         libnss-resolve
         libnss-systemd
         libpam-systemd
-        libsystemd-dev
         libsystemd-shared
         libsystemd0
-        libudev-dev
         systemd
         systemd-container
         systemd-coredump
         systemd-cryptsetup
-        systemd-dev
```

Comment:
Please leave this one, it just has the system pkgconfig files and no dependencies and it's tiny